ios/launch-image-universal.storyboard

```diff
@@ -32,6 +32,6 @@
         </scene>
     </scenes>
     <resources>
-        <image name="launch_image" width="90" height="101"/>
+        <image name="launch_image" width="164" height="169"/>
```

Comment:
hey I noticed this here, is that the Status logo that gets shown on splash screen? why is it not a square 1:1 format, different weight - height?

Reply:
dunno how it works, xcode always generates lots of magic staff, but the logo looks good on splash screen